src/components/todoFeilter.tsx

```diff
@@ -0,0 +1,64 @@
+import { useContext } from 'react';
+import { ActiveContext, AllContext, CompletedContext } from './filterContext';
+
+export const TodosFilter: React.FC = () => {
+  enum Status {
+    All = 'All',
+    Active = 'Active',
+    Completed = 'Completed',
+  }
+
+  const { isAllSelected, setIsAllSelected } = useContext(AllContext);
+  const { isActiveSelected, setIsActiveSelected } = useContext(ActiveContext);
+  const { isCompletedSelected, setIsCompletedSelected } =
+    useContext(CompletedContext);
```

Comment:
it can be only one StatusContext or just one status state that will save the enum Status values

Reply:
there is no any commont bond between those Contexts and enum
contexts just toggle (true or false) for out fiilters.